CMakeLists.txt

```diff
@@ -613,45 +726,35 @@ install(FILES
   )
 if (LAPACK++)
   install(
-  DIRECTORY "${LAPACK_BINARY_DIR}/lib/"
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  FILES_MATCHING REGEX "liblapackpp.(a|so)$"
+    DIRECTORY "${LAPACK_BINARY_DIR}/lib/"
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    FILES_MATCHING REGEX "liblapackpp.(a|so)$"
   )
   install(
-    DIRECTORY "${PROJECT_BINARY_DIR}/lapackpp-prefix/src/lapackpp/include/"
+    DIRECTORY "${lapackpp_BINARY_DIR}/include/"
     DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
     FILES_MATCHING REGEX "\\.(h|hh)$"
   )
-  write_basic_package_version_file(
-    "lapackppConfigVersion.cmake"
-    VERSION 2020.10.02
-    COMPATIBILITY AnyNewerVersion
-  )
   install(
-    FILES "${CMAKE_CURRENT_BINARY_DIR}/lib/lapackpp/lapackppConfig.cmake"
-          "${CMAKE_CURRENT_BINARY_DIR}/lib/lapackpp/lapackppConfigVersion.cmake"
+    FILES "${CMAKE_CURRENT_BINARY_DIR}/lib/cmake/lapackpp/lapackppConfig.cmake"
+          "${CMAKE_CURRENT_BINARY_DIR}/lib/cmake/lapackpp/lapackppConfigVersion.cmake"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/"
   )
 
 endif()
 if (BLAS++)
```

Comment:
LAPACK++ requires BLAS++, so as above, I think this should be `if (BLAS++ OR LAPACK++)`.

Reply:
With `cmake -D LAPACK++=on ..`, this will build BLAS++ and LAPACK++, but doesn't install `libblaspp.a`. Changing to `if (BLAS++ or LAPACK++)` fixes the issue. Can be merged with `if` block below (line 650).